Sources/armory/renderpath/RenderPathDeferred.hx

```diff
@@ -626,6 +646,12 @@ class RenderPathDeferred {
 		}
 		#end
 
+		#if ((rp_voxelgi_shadows) || (rp_voxelgi_refraction))
+		{
+			path.bindTarget(voxels, "voxels");
+		}
+		#end
+
```

Comment:
Are these changes really part of the change of this PR or are they accidentally committed because of your Voxel GI PR?

Reply:
Looks like GitHub didn't like my rather large selection range, this not only refers to the 4 lines that are shown but also to the voxel-related lines above that.

Reply:
No only AO